modules/workbench/modules/monitoring/modules/alert_policies/assets/alert_policies/number_gsuite_users.json

```diff
@@ -0,0 +1,38 @@
+{
+  "combiner": "OR",
+  "conditions": [
+    {
+      "displayName": "GSuite User Count",
+      "conditionThreshold": {
+        "filter": "resource.type = \"generic_node\" AND metric.type = \"custom.googleapis.com/gsuite_user_count\"",
+        "aggregations": [
+          {
+            "alignmentPeriod": "300s",
+            "crossSeriesReducer": "REDUCE_NONE",
+            "perSeriesAligner": "ALIGN_MEAN"
+          }
+        ],
+        "comparison": "COMPARISON_GT",
+        "duration": "0s",
+        "trigger": {
+          "count": 1
+        },
+        "thresholdValue": "${alert_thresholds.gsuite_users_count}"
```

Comment:
Just for my knowledge Where do we define the exact value of alert_thresholds.gsuite_users_count?

Reply:
It will be here https://github.com/all-of-us/workbench-devops/blob/f6c06cb47420cfcb11711b96cb7187a4774a575b/terraform-live/modules/shared/alert_thresholds.tf I'll have another PR to add the values in the different environments